johnsonandjohnson/Bodiless-JS
# Migrate 404 page

## Changes
- Make it possible to not migrate pages with "Page Not Found" (404) status
- Allow an operator to set a specific page that will be used as a template for default 404 page
- Allow disabling above functionality

## How to test
- try migrating some non-existing page
ER: the page should not be migrated, but should redirect to the default /404 page
- add the following option in migration-settings.json
```json
"page404Url": "https://example.com/some-default-404-page"
```
ER: The default 404 page should be created based on the assigned url.
- add the following option in migration-settings.json
```json
"isPage404Disabled": true
```
ER: non-existing pages are not skipped, they are migrated as normal pages.

## Related Issues
Closes #227
